stylingfile.css

```diff
@@ -0,0 +1,178 @@
+
+/* Main Styling */
+
+html, body {
+    background-color: powderblue;
+    box-sizing: border-box;
+    height: 100%;
+}
+
+h1 {
+    color: blue;
+}
+
+p {
+    color: #054bad;
+    font-weight: 500;
+    font-size: 20px;
+}
+
+table,th,td {
+    border: 1px solid black;
+    font-size: 20px;
+    text-align: left;
+}
+
+ul {
+    list-style-type: disc;
+    color: #054bad;
+    font-size: 20px;
+}
+
```

Comment:
I just noticed you don't have any hypertext outside your menu. One instruction was to add a style for hypertexts in general (and to add at least one hypertext on your page). I added a specification for this on Moodle. Can you correct before next lab please?